src/vg/civcraft/mc/citadel/reinforcementtypes/ReinforcementType.java

```diff
@@ -119,6 +122,15 @@ public ItemStack getItemStack(){
 		return stack;
 	}
 	
+	/**
+	 * @param The Material that a player is trying to reinforce.
+	 * @return Returns true if the Material is reinforceable or false otherwise.
```

Comment:
This will return true if the material is found in NonReinforceableTypes; e.g. it returns true if NOT reinforceable. I think your logic was executed right, but the description here is inverted.